packages/openneuro-components/src/facets/FacetSelect.tsx

```diff
@@ -3,15 +3,13 @@ import { AccordionTab } from '../accordion/AccordionTab'
 import { AccordionWrap } from '../accordion/AccordionWrap'
 import './facet.scss'
 
-export interface FacetListWrapProps {
-  items: [
-    {
-      label: string
-      value: string
-      count: number
-      children: React.ReactNode
-    },
-  ]
+export interface FacetSelectProps {
+  items: {
+    label: string
+    value: string
+    count: number
```

Comment:
```suggestion
    count?: number
```
for this line and 22 in `selected`